### ISSUE TITLE
git commit does not work when launched from within WSL

### ISSUE THREAD

**Issue by bramasub1** [microsoft/vscode#179303]
Type: <b>Bug</b>

When vscode is launched from my WSL instance, git commit doesn't work.  Instead of creating a new tab, it errors out as:

> git -c user.useConfigOnly=true commit --quiet
Error: connect EACCES /mnt/wslg/runtime-dir/vscode-git-0730645f99.sock
    at PipeConnectWrap.afterConnect [as oncomplete] (node:net:1157:16) {
  errno: -13,
  code: 'EACCES',
  syscall: 'connect',
  address: '/mnt/wslg/runtime-dir/vscode-git-0730645f99.sock'
}
error: There was a problem with the editor '"/home/bram/.vscode-server/bin/b7886d7461186a5eac768481578c1d7ca80e2d21/extensions/git/dist/git-editor.sh"'.
Please supply the message using either -m or -F option.

VS Code version: Code 1.77.1 (b7886d7461186a5eac768481578c1d7ca80e2d21, 2023-04-04T23:21:11.906Z)
OS version: Windows_NT x64 10.0.19044
Modes:
Sandboxed: No
Remote OS version: Linux x64 5.15.90.1-microsoft-standard-WSL2

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i7-11850H @ 2.50GHz (16 x 2496)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|31.73GB (6.99GB free)|
|Process Argv|--crash-reporter-id 532826a8-fe53-4b7d-88a3-8a820cd6e7db|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: OracleLinux_9|
|OS|Linux x64 5.15.90.1-microsoft-standard-WSL2|
|CPUs|11th Gen Intel(R) Core(TM) i7-11850H @ 2.50GHz (16 x 2496)|
|Memory (System)|15.49GB (13.21GB free)|
|VM|0%|
</details><details><summary>Extensions (20)</summary>

Extension|Author (truncated)|Version
---|---|---
jupyter-keymap|ms-|1.1.0
remote-containers|ms-|0.288.0
remote-ssh|ms-|0.98.0
remote-ssh-edit|ms-|0.84.0
remote-wsl|ms-|0.77.0
vscode-remote-extensionpack|ms-|0.24.0
remote-explorer|ms-|0.4.0
remote-server|ms-|1.0.0
doxdocgen|csc|1.4.0
gitlens|eam|13.4.0
vscode-clangd|llv|0.1.23
python|ms-|2023.6.0
vscode-pylance|ms-|2023.3.40
jupyter|ms-|2023.3.1000892223
jupyter-renderers|ms-|1.0.15
vscode-jupyter-cell-tags|ms-|0.1.8
vscode-jupyter-slideshow|ms-|0.1.5
cpptools|ms-|1.14.5
makefile-tools|ms-|0.6.0
clang-format|xav|1.9.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscorecescf:30445987
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vscaac:30438847
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
pynewext54:30695312
azure-dev_surveyone:30548225
nodejswelcome1cf:30587006
3biah626:30602489
pyind779:30671433
89544117:30613380
pythonsymbol12:30671437
2i9eh265:30646982
vsccsb:30705552
pythonb192cf:30669361
functionswalk:30687959
pythonms35:30701012
19089226:30706075
pythonclientmv:30705673

```

</details>

**Comment by lszomoru** [comment]
@bramasub1, are you able to commit if you try doing so from the Terminal (WSL), of the VS Code integrated terminal? Are you able to reproduce this error with more than one repository or is it specific to one specific repository?

**Comment by bramasub1** [comment]
1) Yes, I am able to commit from both WSL terminals and the vscode integrated terminal.  
2) Commit also works if I enter a message in the box above the commit button.  It doesn't work if the commit message is empty. I expect it to open a new tab in vscode to let me enter the commit message
3) It doesn't work for any of my repositories.

**Comment by tabish-javed** [comment]
Exact same problem returned on VSCode November 23 release 1.84.1 inside WSL2. Has anyone resolved by any tweak?

**Comment by bramasub1** [comment]
This has been open for a while.  Any update ?

**Comment by bramasub1** [comment]
Looks like this has been fixed in version 1.89 and can be closed now.
